package.json

```diff
@@ -21,13 +21,14 @@
   },
   "homepage": "https://github.com/holepunchto/hyperdrive-next#readme",
   "dependencies": {
-    "hyperbee": "^2.1.1",
+    "hyperbee": "^2.8.0",
```

Comment:
Would bump to latest version (I think you're already using the API of later versions here, like specifying a keyEncoding for the watch)